timemachine/fe/single_topology.py

```diff
@@ -896,7 +896,7 @@ def combine_masses(self):
 
         return mol_c_masses
 
-    def combine_confs(self, x_a, x_b):
+    def combine_confs(self, x_a, x_b, lamb=1.0):
```

Comment:
nit: document `lamb`

Reply:
Documented in https://github.com/proteneer/timemachine/pull/936/commits/3afbf08735710f4eeba52fd5569f95c5fdfda77a